proto/regen/group/v1alpha1/query.proto

```diff
@@ -0,0 +1,246 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+import "regen/group/v1alpha1/types.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+// Query is the regen.group.v1alpha1 Query service.
+service Query {
+
+  // GroupInfo queries group info based on group id.
+  rpc GroupInfo(QueryGroupInfoRequest) returns (QueryGroupInfoResponse);
```

Comment:
It can be a separate PR but grpc-gateway annotations are missing.

Reply:
yes, cf. https://github.com/regen-network/regen-ledger/issues/182